scripts/provision.sh

```diff
@@ -35,19 +51,23 @@ EOT
 
 configure() {
   export PGPASSWORD PGUSER
-
-  if [ "${RDS_ENABLED:-false}" = "false" ]; then
-    while [ ! -f /hab/svc/builder-datastore/config/pwfile ]
-    do
-      sleep 2
-    done
-
-    PGUSER='hab'
-    PGPASSWORD=$(cat /hab/svc/builder-datastore/config/pwfile)
-  else
-    PGUSER=${RDS_USER:-hab}
-    PGPASSWORD=${RDS_PASSWORD:-hab}
-  fi
+    if [ "${RDS_ENABLED:-false}" = "false" ]; then
```

Comment:
Should this be `==`?

Reply:
Nope. As I understand it, the `[]` in bash are a synonym for `test` and for the `test` syntax this is correct.

Reply:
That makes a ton more sense now.  Thanks for connecting those dots for me 😸